cli/src/file_utils.rs

```diff
@@ -42,9 +42,15 @@ pub fn get_files(directory: &str, paths_to_ignore: &[String]) -> Result<Vec<Path
 
         // check if the path should be ignored by a glob or not.
         for path_to_ignore in paths_to_ignore {
-            // we build the expanded glob to make sure it will match with the full path.
-            let expanded_glob = format!("{}/{}", directory, path_to_ignore);
-            if glob_match(expanded_glob.as_str(), entry.display().to_string().as_str()) {
+            let path_buf = entry.to_path_buf();
```

Comment:
Why not introducing a short-circuit if `should_include` is already `false` in ln.41:
```rust
 if !should_include {
    continue;
}
```

Reply:
The reason is because I want to favor simplicity/readability over performance, especially for this part that is here only in the beginning.

Reply:
In my opinion, adding the short-circuit here doesn't hurt readability giving that it's just 3 lines just after `should_include` definition. On the contrary, it makes clear the intent. At least for me, not having that short-circuit make the code a little bit less intuitive. I had to inspect the whole block to understand the motivation because I found odd not having the short-circuit in there in the first place. 

That being said, it's just a suggestion and some of these things are subjective. I added the comment because of the reasons exposed before (totally subjective) and for performance (small in this case).